lib/Swagger/src/Discussion/Models/VotingResponse.php

```diff
@@ -0,0 +1,289 @@
+
+use \ArrayAccess;
+
+/**
+ * VotingResponse Class Doc Comment
+ *
+ * @category    Class */
+/** 
+ * @package     Swagger\Client
+ * @author      http://github.com/swagger-api/swagger-codegen
+ * @license     http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @link        https://github.com/swagger-api/swagger-codegen
+ */
+class VotingResponse implements ArrayAccess
+{
+    /**
+      * The original name of the model.
+      * @var string
+      */
+    protected static $swaggerModelName = 'VotingResponse';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
+      * @var string[]
+      */
+    protected static $swaggerTypes = array(
+        'requester_id' => 'string',
+        'upvote_count' => 'int',
+        '_links' => 'map[string,\Swagger\Client\Discussion\Models\NewHalLink]'
+    );
+
+    public static function swaggerTypes()
+    {
+        return self::$swaggerTypes;
+    }
+
+    /**
+     * Array of attributes where the key is the local name, and the value is the original name
+     * @var string[]
+     */
+    protected static $attributeMap = array(
+        'requester_id' => 'requesterId',
+        'upvote_count' => 'upvoteCount',
+        '_links' => '_links'
+    );
+
+    public static function attributeMap()
+    {
+        return self::$attributeMap;
+    }
+
+    /**
+     * Array of attributes to setter functions (for deserialization of responses)
+     * @var string[]
+     */
+    protected static $setters = array(
+        'requester_id' => 'setRequesterId',
+        'upvote_count' => 'setUpvoteCount',
+        '_links' => 'setLinks'
+    );
+
+    public static function setters()
+    {
+        return self::$setters;
+    }
+
+    /**
+     * Array of attributes to getter functions (for serialization of requests)
+     * @var string[]
+     */
+    protected static $getters = array(
+        'requester_id' => 'getRequesterId',
+        'upvote_count' => 'getUpvoteCount',
+        '_links' => 'getLinks'
+    );
+
+    public static function getters()
+    {
+        return self::$getters;
+    }
+
+    
+
+    
+
+    /**
+     * Associative array for storing property values
+     * @var mixed[]
+     */
+    protected $container = array();
+
+    /**
+     * Constructor
+     * @param mixed[] $data Associated array of property value initalizing the model
+     */
+    public function __construct(array $data = null)
+    {
+        $this->container['requester_id'] = isset($data['requester_id']) ? $data['requester_id'] : null;
+        $this->container['upvote_count'] = isset($data['upvote_count']) ? $data['upvote_count'] : null;
+        $this->container['_links'] = isset($data['_links']) ? $data['_links'] : null;
+    }
+
+    /**
+     * show all the invalid properties with reasons.
+     *
+     * @return array invalid properties with reasons
+     */
+    public function listInvalidProperties()
+    {
+        $invalid_properties = array();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$invalid_properties". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)